src/guest_memory.rs

```diff
@@ -77,6 +77,11 @@ pub enum Error {
     /// Host virtual address not available.
     #[error("Guest memory error: host virtual address not available")]
     HostAddressNotAvailable,
+    /// The length returned by the callback passed to `try_access` is outside the address range.
+    #[error(
+        "The length returned by the callback passed to `try_access` is outside the address range."
+    )]
+    CallbackOutOfRange,
```

Comment:
Lets reuse `Error::InvalidGuestAddress` here, so that this can be a patch release

Reply:
Updated to use this approach.